pkg/dir.props

```diff
@@ -28,7 +28,7 @@
   <PropertyGroup>
     <PackageOutputPath>$(PackagesOutDir)</PackageOutputPath>
     <SymbolPackageOutputPath>$(SymbolPackagesOutDir)</SymbolPackageOutputPath>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.2\runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\DEPENDENCY 'Microsoft.NETCore.Platforms' NOT FOUND\runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
Well, the auto-upgrade in core-setup looks for that package by name in https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt, and since it didn't find it, it added a noticeable string. This happens around [UpdateFilesTargets.cs#L234](https://github.com/dotnet/core-setup/blob/4c41eace622faf3cdfdd0bcd4030a71235fbd3d5/build_projects/update-dependencies/UpdateFilesTargets.cs#L234).

A fix would be to condition that replace on whether or not a version is found.